pkg/apis/core/types_shoot.go

```diff
@@ -1044,6 +1044,11 @@ type KubeletConfig struct {
 	StreamingConnectionIdleTimeout *metav1.Duration
 	// MemorySwap configures swap memory available to container workloads.
 	MemorySwap *MemorySwapConfiguration
+	// MaxParallelImagePulls describes the maximum number of image pulls in parallel. The value must be a positive number.
+	// When updating this field, the SerializeImagePulls (pull one image at a time) must be set to false.
```

Comment:
Nit: A wording suggestion inspired from https://github.com/kubernetes/kubernetes/blob/82db38a23c7820b1924d89f458fd368023f3980c/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L544C5-L544C61.

```suggestion
	// This field cannot be set if SerializeImagePulls (pull one image at a time) is set to true.
```

Reply:
Thank you for the suggestion and for the review!